ppl/zqd/core.go

```diff
@@ -147,7 +140,14 @@ func NewCore(ctx context.Context, conf Config) (*Core, error) {
 	return c, nil
 }
 
-func (c *Core) addAPIServerRoutes() {
+func (c *Core) addAPIServerRoutes(ctx context.Context, conf Config) (err error) {
+	c.root, err = iosrc.ParseURI(conf.Root)
+	if err != nil {
+		return
+	}
+	if c.mgr, err = apiserver.NewManager(ctx, conf.Logger, c.registry, c.root, conf.DB); err != nil {
+		return
```

Comment:
Nit: I think adorned returns improve readability.

Reply:
I can get behind this.